nxviz/annotate.py

```diff
@@ -46,15 +46,23 @@ def text_alignment(x, y):
     return ha, va
 
 
+def validate_fontdict(fontdict: Dict):
+    """Validate `fontdict` keys."""
+    valid_keys = {"family", "size", "stretch", "style", "variant", "weight"}
+    assert set(fontdict) <= valid_keys
```

Comment:
First time I've seen this syntax! Is it checking that `set(fontdict)` is a subset of `valid_keys`?

Reply:
Yes.

Reply:
Cool stuff. I'm learning stuff from you too :smile:.